### PR TITLE
UI Build out

### PR DESCRIPTION
**Pull request by teenyrobots**
This branch builds out the user interface and adds real content

### CODE REVIEW

File: data/students.json [hunk 1]
```diff
@@ -1,75 +1,314 @@
 {
   "students": [
     {
-      "displayName": "Guillory Martin",
+      "displayName": "Trinidad Escobar",
+      "slug": "escobar",
+      "year": 2015,
```
Comment (kevmo) [inline data/students.json]: I would recommend changing all the years to strings (as they were originally were).  Less potential for bugs, I suspect.

File: data/students.json [hunk 2]
```diff
@@ -1,75 +1,314 @@
...
+      "thesisPath": "15white/"
+    },
+    {
+      "displayName": "Shane McDermott",
+      "slug": "mcdermott",
+      "year": 2015,
+      "thesisPages": 14,
+      "thesisTitle": "The Seahorse Adventures",
+      "digitalTitle": "",
+      "digitalUrl": "",
```
Comment (kevmo) [inline data/students.json]: no data here

File: data/students.json [hunk 3]
```diff
@@ -1,75 +1,314 @@
...
+      "thesisPath": "16morehead/"
+    },
+    {
+      "displayName": "Kristin Olson",
+      "slug": "olson",
+      "year": 2016,
+      "thesisPages": 6,
+      "thesisTitle": "Sick Bed Blues",
+      "digitalTitle": "The Crystal Ball",
+      "digitalUrl": "16olson/pg1.html",
```
Comment (kevmo) [inline data/students.json]: Why are some digitalUrls directories and others files?

File: data/students.json [hunk 4]
```diff
@@ -1,75 +1,314 @@
...
+      "slug": "ramm",
+      "year": 2017,
+      "thesisPages": 16,
+      "thesisTitle": "TK",
+      "digitalTitle": "EXPLORE THE INTERNET",
+      "digitalUrl": "17ramm/",
+      "thesisPath": "17ramm/"
     }
   ]
 }
```
Comment (kevmo) [inline data/students.json]: Add a newline at end of file.

File: directory.html [hunk 5]
```diff
@@ -6,98 +6,107 @@
...
 <body>
 
 
-
     <noscript>
         You need to enable JavaScript to run this app.
     </noscript>
 
-
+<div class="transp">
```
Comment (kevmo) [inline directory.html]: I recommend spelling the word out ("transparent"?).  Clarity is much more important than saving a few bytes.

File: index.css [hunk 6]
```diff
@@ -61,39 +98,31 @@ body {
...
 }
 
 #bigHeader h3 {
     font-weight: 300;
 }
 
 #stickyHeader {
+    display: block;
+    margin: auto;
+    width: 100%;
```
Comment (kevmo) [inline index.css]: I use `100vw` a lot these days.

File: index.css [hunk 7]
```diff
@@ -104,45 +133,103 @@ body {
...
-  margin-bottom: 1em;
-
+  margin-bottom: .5em;
 }
 
+ i stopped using a person image
 .person img {
   border-radius: 100%;
   float: left;
   margin: 0 1em;
```
Comment (kevmo) [inline index.css]: I would delete this commented code now.

**Review by kevmo — COMMENTED** [review]
Couple of salient questions on the `students.json`.  

Also made a few quick comments.

I recommend manually reading through students JSON and thinking while you do it.  Would recommend changing years to strings while doing it.

File: data/students.json (hunk 3 above)
Comment (kevmo) [reply data/students.json]: this is Q i am most interested in.

File: data/students.json (hunk 1 above)
Comment (teenyrobots) [reply data/students.json]: 👍🏻

File: data/students.json (hunk 2 above)
Comment (teenyrobots) [reply data/students.json]: yeah so there are couple people who don't have digital projects and i think they're just going to be last in the lineup for their year and i'll write some logic to not display anything if there's no data

File: data/students.json (hunk 3 above)
Comment (teenyrobots) [reply data/students.json]: when there's an index.html file i just linked to the directory...if that's bad form i can just paste in the "index.html" that's nbd

File: data/students.json (hunk 4 above)
Comment (teenyrobots) [reply data/students.json]: 👍🏻

File: directory.html (hunk 5 above)
Comment (teenyrobots) [reply directory.html]: 👍🏻

File: index.css (hunk 6 above)
Comment (teenyrobots) [reply index.css]: 👍🏻

File: index.css (hunk 7 above)
Comment (teenyrobots) [reply index.css]: 👍🏻

File: data/students.json [hunk 8]
```diff
@@ -1,75 +1,314 @@
...
+      "thesisPages": 16,
+      "thesisTitle": "TK",
+      "digitalTitle": "EXPLORE THE INTERNET",
+      "digitalUrl": "17ramm/index.html",
+      "thesisPath": "17ramm/"
     }
   ]
-}
\ No newline at end of file
+}
```
Comment (kevmo) [inline data/students.json:314]: This looks like a more complete, reliable data structure.

Comment (kevmo) [reply data/students.json:314]: https://en.wikipedia.org/wiki/Single_source_of_truth

**Review by kevmo — APPROVED** [review]
This looks good to me.  Dropped SSOT wikipedia link for some educational background.

**Pull request merged**